docs/tutorials/solar_flares.ipynb

```diff
@@ -844,45 +1062,51 @@
    "cell_type": "markdown",
    "id": "61d012c3-76f6-41e8-9294-686bad746738",
    "metadata": {
+    "jp-MarkdownHeadingCollapsed": true,
     "tags": []
    },
    "source": [
-    "---\n",
-    "<details>\n",
-    "<summary class=\"solution\">\n",
-    "  Solution\n",
-    "</summary>\n",
-    "\n",
-    "<br>\n",
-    "Bin in time and keep the existing spatial binning.\n",
-    "`transpose` is used to put the time dimension on the slider in the plot.\n",
-    "\n",
-    "```python\n",
-    "temporal_and_spatial = spatial.bin(peak_time=200)\n",
-    "to_plot = temporal_and_spatial.hist()\n",
-    "to_plot.transpose(['peak_time', 'y', 'x']).plot(vmin=0.0 * sc.units.one,\n",
-    "                                                vmax=0.66 * sc.units.one,\n",
-    "                                                labels={\n",
-    "                                                    'x': 'x [asec]',\n",
-    "                                                    'y': 'y [asec]'\n",
-    "                                                },\n",
-    "                                                cmap='inferno')\n",
-    "```\n",
-    "\n",
-    "<br>\n",
-    "Or remove the existing binning, yielding a 1D-variable.\n",
-    "\n",
-    "```python\n",
+    "#### Solution"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0f5fa729-b782-4e02-9545-bd0bf5a352b4",
+   "metadata": {},
+   "source": [
+    "Remove the existing binning and bin in time to yield a 1D-variable."
```

Comment:
```suggestion
    "Remove the existing binning and bin in time to yield a 1D-variable representing the number of flares in each time bin."
```
which could maybe remove the need for having `p.canvas.ylabel = 'number of flares'` multiple times below?
(I think the unit would then just be counts)

Reply:
What was the change that removed the need for having `ylabel = 'number of flares'`?

Reply:
That we say 'number of flares' in the text, so that we don't have to write it on the figures and use the default 'counts' instead.